### PR TITLE
Update i18n example test to actually exercise vuei18n

### PR DESCRIPTION
**Pull request by sarayourfriend**
This shows a minimal reproduction of the issue discussed in #585

### CODE REVIEW

File: examples/i18n/app.vue [hunk 1]
```diff
@@ -1,5 +1,14 @@
 <template>
   <div>
-    Hi from @nuxtjs/i18n: {{ (('hello')) }}
+    Hi from @nuxtjs/i18n: {{ $t('hello') }}
+  </div>
+  <div>
```
Comment (obulat) [inline examples/i18n/app.vue:5]:
```suggestion
  <i18n-t scope="global" keypath="message"><template #module>@nuxtjs/i18n</template></i18n-t>
  <div>
```
The error in i18n _can_ be fixed by importing `i18n` from `useNuxtApp().$i18n`. However, I couldn't find a way to fix the `i18n-t` component. I tried replacing it with the `Translation` component imported from `vue-i18n`, but the error persists.

This suggested change will also need an addition to the `i18n` messages:
`"message": "Hello from {module}"`

**Comment by danielroe** [comment]
As far as I can tell the issue here is because the `@nuxtjs/i18n` module is calling `vueApp.use` on the CJS version of `vue-i18n`, but when we use `useI18n` in the test, it's the ESM version of the library, and so it thinks it's not registered.

Without looking further (though of course I will) I'm not sure whether this is something that `@nuxt/test-utils` should solve or whether it's an upstream issue ....

cc: @BobbieGoede so you're aware

**Comment by sarayourfriend** [comment]
Thanks for figuring that out! Indeed, that doesn't seem like something this library should fix. @obulat I wonder if we can force Openverse (or at least the test suite) to use the CJS version?

**Comment by obulat** [comment]
I opened an issue with the reproductions (using composable and `i18n-t` component) in the `@nuxt-modules/i18n` repository.
@sarayourfriend, I tried to find a way to hot-fix this issue, but I couldn't. 
@danielroe, thank you for investigating this issue. Do you have any suggestions for hot-fixing this until the solution is found?

**Comment by danielroe** [comment]
I still plan to investigate this further.

**Comment by indiehjaerta** [comment]
Any updates on the investigation of this? As it stands now this problem stops us from upgrading to the  @nuxt/test-utils from nuxt-vitest that we have used before.

**Pull request closed without merging**